tex/measure/measure-space.tex

```diff
@@ -238,7 +238,7 @@ \section{Measure spaces}
 		\ii The axioms on $\mu$ can be weakened to
 		(i) $\mu(\varnothing) = 0$,
 		(ii) $\mu(A \sqcup B) = \mu(A) + \mu(B)$, and
-		(iii) for $A_1 \supset A_2 \supset \dots$
+		(iii) for $A_1 \subset A_2 \subset \dots$
```

Comment:
I think \supset was correct? Since it's \bigcap after.

Reply:
Although I suppose I should be using `\supseteq` anyways to be consistent with the rest of the book.

Reply:
Hmm, yeah, whoops, let's use \supseteq.